types/parse/index.d.ts

```diff
@@ -727,6 +727,15 @@ declare global {
         }
         const Installation: InstallationConstructor;
 
+        // Read preference describes how MongoDB driver route read operations to the members of a replica set.
+        enum ReadPreferenceOption {
+            Primary = "PRIMARY",
+            PrimaryPreferred = "PRIMARY_PREFERRED",
+            Secondary = "SECONDARY",
+            SecondaryPreferred = "SECONDARY_PREFERRED",
+            Nearest = "NEAREST",
+        }
```

Comment:
Does this enum actually exist in the code (i.e. is there an object called `ReadPreferenceOption` somewhere)?